R/temp.R

```diff
@@ -40,6 +42,9 @@ env$temp_names <- character()
 file_temp <- function(pattern = "file", tmp_dir = tempdir(), ext = "") {
   assert_no_missing(tmp_dir)
 
+  prepend_dot <- function(ext) ifelse(nchar(ext), paste0(".", ext), ext)
```

Comment:
This likely does not matter much for this case, but for clarity's sake it is probably worth using the vectorization of `nzchar()` and `paste0()`. e.g.

```r
has_extension <- !nzchar(ext)
ext[has_extension] <- paste0(".", ext[has_extension])
```

Reply:
Ah, cool! I didn't know about `nzchar()`, that is more clear for sure. I removed the `!` because it inverted `has_extension`. Thanks for the tip!